doc/overview.rst

```diff
@@ -29,13 +29,30 @@ rosdep gets its data from.
 ``rosdep update`` reads through this sources list to initialize your
 local database.
 
+You can override the location by setting environment variable
+``ROSDEP_SOURCE_PATH``. The custom path has to exist prior to calling
+``rosdep init``, otherwise the default one will be used.
+
+Please note that when using ``sudo``, environment
+variables from the user are not passed to the command. To specify the variable
+for initializing the database, call::
+
+    sudo mkdir -p /usr/rosdep.sources
+    sudo env ROSDEP_SOURCE_PATH=/usr/rosdep.sources rosdep init
```

Comment:
`--preserve-env=` is another method for passing environment context to sudo commands.

```suggestion
Please note that when using ``sudo``, environment
variables from the user are not passed to the command. To specify the variable
for initializing the database, call::

    sudo mkdir -p /usr/rosdep.sources
    sudo env ROSDEP_SOURCE_PATH=/usr/rosdep.sources rosdep init
    
Alternatively you can use `--preserve-env` to pass through current environment variable values.
    export ROSDEP_SOURCE_PATH=/tmp/example-rosdep-sources
    mkdir $ROSDEP_SOURCE_PATH
    sudo --preserve-env=ROSDEP_SOURCE_PATH rosdep init
```